packages/cli/src/build-stylable.ts

```diff
@@ -81,6 +82,10 @@ export async function buildStylable(
 
             log('[Project]', projectRoot, buildOptions);
 
+            if (!hasStylableCSSOutput(buildOptions)) {
+                log(`No CSS output found for "${identifier}"`, levels.info);
```

Comment:
Perhaps consider changing to something like:
`No target output declared, please provide one or more of the following target options: "cjs", "esm", "css", "stcss" or "indexFile"`

Reply:
and possibly a higher severity?